x-pack/test/reporting_api_integration/reporting_and_security/diagnostic/screenshot.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import assert from 'assert';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+interface ScreenshotResponse {
+  logs: string[];
+  capture: string;
+  help: string[];
+  success: true;
+}
+
+// eslint-disable-next-line import/no-default-export
+export default function ({ getService }: FtrProviderContext) {
+  const supertest = getService('supertestWithoutAuth');
+
+  describe('browser', () => {
+    const apiPath = '/api/reporting/diagnose/screenshot';
+    const username = 'elastic';
+    const password = process.env.TEST_KIBANA_PASS || 'changeme';
+
+    let body: ScreenshotResponse;
+    let status: number;
+
+    before(async () => {
+      const response = await supertest
+        .post(apiPath)
+        .auth(username, password)
+        .set('kbn-xsrf', 'xxx');
+      ({ body, status } = response);
+    });
+
+    it('sends captured screenshot', async () => {
+      assert(body.capture);
+    });
+
+    it('sends screenshotting logs', async () => {
+      const logs = body.logs.join();
+      expect(logs).contain(`Creating browser page driver...`);
+      expect(logs).contain(`Chromium launch args set to`);
+      expect(logs).contain(`Sandbox is disabled`);
+      expect(logs).contain(`Launching with viewport: width=800 height=600 scaleFactor=1`);
+      expect(logs).contain(`Operating system`);
+      expect(logs).contain(`Browser version`);
+      expect(logs).contain(`Setting browser timezone to UTC...`);
+      expect(logs).contain(`Browser page driver created`);
+      expect(logs).contain(`Navigating URL with viewport size: width=800 height=600 scaleFactor:1`);
+      expect(logs).contain(`evaluate WaitForElementsToBeInDOM: waitng for 2 elements, got 2`);
+      expect(logs).contain(`Chromium consumed`);
```

Comment:
Notes:
* In the actual logs, all of these messages have timestamps, which could not be tested against.
* There is not a log message for `Capturing screenshot...` but it can be implied that it happens just before we log the CPU/Memory metrics.